src/components/Captcha.vue

```diff
@@ -166,6 +166,7 @@ export default {
   created() {
     this.fetchNewCaptcha();
   },
+  emits: ['captchaLoaded', 'captchaVerified'],
```

Comment:
When there's more than one element in an array, I usually like to break it up so each element has its own line. So here it would be be:
```javascript
emits: [
  'captchaLoaded',
  'captchaVerified'
],
```
There's a couple other places in this PR where this occurred as well.